src/data/en/index.ts

```diff
@@ -354,18 +358,23 @@ export const instanceData = {
         1060001: 'You successfully recovered your account. Please change your password in the next minutes.',
         1060002: 'An email containing a recovery link has been sent to the email address you provided.',
         1080001: 'An email containing a verification link has been sent to the email address you provided.',
-        1080002: 'You successfully verified your email address.',
+        1080002: emailVerifiedSuccessfully,
         4000001: '%reason%',
-        4000002: 'Property %field% is missing.',
+        4000002: '%field% is missing.',
         4000003: 'Length must be \u003e= %expected_length%, but got %actual_length%.',
         4000004: '"%value%" is not valid "%format%"',
```

Comment:
side note: we never replace the placeholders here and nobody saw those strings while testing so we can maybe remove them.